src/core/agent/services/keriaNotificationService.ts

```diff
@@ -813,11 +813,11 @@ class KeriaNotificationService extends AgentService {
       }
       case OperationPendingRecordType.Oobi: {
         const connectionRecord = await this.connectionStorage.findById(
-          (operation.response as any).i
+          (operation.response as State).i
         );
         if (connectionRecord) {
           connectionRecord.pending = false;
-          connectionRecord.createdAt = (operation.response as any).dt;
+          connectionRecord.createdAt = new Date((operation.response as State ).dt);
```

Comment:
Shouldn't we be calling `contacts().update()` here? update, and add test

Reply:
No, originally we didn't call contacts().update() here. I misunderstood what you meant, so in the previous commit I placed the logic of calling update in resolveOobi() here.

Reply:
We need to call it here. If we create a normal connection where we use the long running operation, once it completes we need to call `contacts().update()`. As I said, once my PR merges for keripy it won't be auto-created like it is now on KERIA, so the contact would not be on KERIA with this current PR.

Reply:
Yes, so when we resolve a normal connection OOBI (not a schema, not a multi-sig setup QR code), we will always set it as pending and create a long running operation to track the OOBI resolution.

When the OOBI resolution completes (where this comment is), we will call `contacts().update()`. `contacts().update()` won't work until the OOBI has been resolved so we have to do it here.

Reply:
@Sotatek-TungNguyen2a you need to pass the `alias` in the long running operation record.

Otherwise, the `randomSalt` will be different! Or if the user selects it (which the UI will be doing soon), it won't have it from here.

Reply:
oh, you mean use alias in ConnectionRecord